src/app/screens/Send/index.tsx

```diff
@@ -66,6 +67,8 @@ function Send() {
         }
 
         if (lnurlDetails.tag === "payRequest") {
+          originData.name = lnurlDetails.domain;
```

Comment:
No sure if this is wanted. If we do not want this we could remove the origin completely here (and checking if this is used in other places i.e. `payment.location` etc)